cmd/egress/cmd.go

```diff
@@ -13,8 +13,8 @@ import (
 	"time"
 
 	"github.com/openshift/osd-network-verifier/cmd/utils"
+	platform "github.com/openshift/osd-network-verifier/pkg/data/cloud"
```

Comment:
Just a thought: any reason why we have to stick to `platform`? Would just keeping the name `cloud` be too weird?